assets/scss/02-molecules/_relationship-indicators.scss

```diff
@@ -201,5 +136,85 @@
                 color: $c-theme-blue;
             }
         }
+
+        a {
+            display: block;
+            border: 2px solid;
+            padding: 1px 5px;
+            line-height: 1.2;
+            width: 100%;
+        
+            @media ($bp-x-small-min) {
+                width: auto;
+                display: inline-block;
+            }
+        }
+
+        button {
+            @include ma-button-reset;
+            @include ma-link-underline;
+            @include ma-chevron;
+            text-transform: capitalize;
+            align-self: flex-start;
+            margin-top: 5px;
+            padding-top: 0;
+            padding-left: 0;
+            font-size: 1rem;
+            font-weight: 600;
+            white-space: nowrap;
+            display: none;
+            
+            @media ($bp-small-min) {
+                margin-top: 3px;
+            }
+            
+            @media ($bp-medium-min) {
+                align-self: baseline;
+                margin-top: 8px;
+            }
+            
+            .tag-state {
+                text-transform: none;
+            }
+            
+            &:after {
```

Comment:
Next time you are in this file, we need to change all pseudo elements to use the double `::` and not the single `:`